pkg/command/suspend/cmd.go

```diff
@@ -73,6 +73,10 @@ func (cmd *command) init() {
 func (cmd *command) run(args []string) error {
 	// On plain suspend call do not create a task.
 	if !cmd.Update() && cmd.duration.Value() == 0 {
+		if cmd.startTasks {
```

Comment:
I think you are abusing the existing code structure.

This should also affect the updates.
Add validation, if not valid return error.

Reply:
I think `school suspend update` in general is a bit more problematic and needs to be either removed or reworked. Most of the things you can update actually do nothing.

For example `school suspend update suspend/xyz --duration 1h` does nothing. Whether it is performed on the actual suspend task, or the task `suspend/resume`. Most of the other flags available to update a suspend task are equally non-functioning or unnecessary.

```
Usage:
  sctool suspend update --cluster <id|name> [flags] [<suspend/task-id>]

Flags:
  -c, --cluster name or ID       The target cluster name or ID (envvar SCYLLA_MANAGER_CLUSTER).
                                  (default "my-cluster")
      --cron expression          Task schedule as a cron expression.
                                 It supports the extended syntax including @monthly, @weekly, @daily, @midnight, @hourly, @every X[h|m|s].

      --duration duration        Automatically resume after the given duration X[h|m|s].
      --enabled                  Not enabled tasks are not executed and hidden from task list.
                                  (default true)
  -h, --help                     help for update
      --name string              Task name that can be used insead of ID.

  -r, --num-retries int          Number of times a task reruns following a failure.
                                  (default 3)
      --on-resume-start-tasks    On resume start tasks that were stopped by the suspend.
      --retry-wait duration      Initial exponential backoff duration X[h|m|s].
                                 With --retry-wait 10m task will wait 10 minutes, 20 minutes and 40 minutes after first, second and third consecutire failure.
                                  (default "10m")
      --timezone string          Timezone of --cron and --window flag values.
                                 The default value is taken from this system, namely 'TZ' envvar or '/etc/localtime' file.
                                  (default "Europe/Amsterdam")
      --window [WEEKDAY-]HH:MM   A comma-separated list of time markers in a form [WEEKDAY-]HH:MM.
                                 WEEKDAY can be written as the whole word or only using the first 3 characters, HH:MM is an hour from 00:00 to 23:59.

                                 'MON-00:00,FRI-15:00' - can be executed from Monday to Friday 3PM
                                 '23:00,06:00' - can be executed every night from 11PM to 6AM
                                 '23:00,06:00,SAT-00:00,SUN-23:59' - can be executed every night from 11PM to 6AM and all day during the weekend
```

The only one that I think really would provide any value is changing the duration so you can update when the manager resumes.

Reply:
Misunderstood this a bit, now I see how the update is working and useful for the scheduled suspends. The issue is just that you can do `sctool update suspend/resume` which appears successful but is completely ineffective.

Will add an error message when that is attempted as well.